### PR TITLE
OSIDB-2743: Remove  Affect type

### PR DESCRIPTION
**Pull request by MrMarble**
Removed the `AffectType` enum and the `type` field from the `Affect` model as it only has one possible value and is not needed at all.

Closes OSIDB-2743

### CODE REVIEW

**Review by osoukup — APPROVED** [review]
LGTM

**Review by JakubFrejlach — CHANGES_REQUESTED** [review]
I would update the `CHANGELOG.md` this is field removal and it directly influences API so it is something which potential user would like to see in the changelog. Otherwise LGTM!

**Review by JakubFrejlach — APPROVED** [review]

**Review by dmonzonis — APPROVED** [review]

File: osidb/migrations/0129_remove_affect_type.py
Comment (jobselko) [inline osidb/migrations/0129_remove_affect_type.py:1]: There already is migration `0129_delete_flawhistory.py`, so you have to create a new one.

Comment (MrMarble) [reply osidb/migrations/0129_remove_affect_type.py:1]: Oh yeah, I did a rebase because of that and totally forgot to rename the migration file, changed

**Pull request merged**